protoc_plugins/protoc-gen-php-grpc/php/ns.go

```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"fmt"
 	"strings"
-
+	
```

Comment:
Watch for whitespace inconsistencies like this, especially when submitting to external repos.